### PR TITLE
Feature/us11316 integrate server side

### PR DESCRIPTION
**Pull request by Ganonside**

### CODE REVIEW

File: src/home/home.js [hunk 1]
```diff
@@ -1,17 +1,44 @@
...
+      return;
+    }
+
+    const data = {
+      firstName,
+      lastInitial,
+      prayer,
+    };
+
+    this.formioService.submit(file, fileName, file.size, file.type, data);
```
Comment (chris-branch) [inline src/home/home.js]: Parameters passed here do not match function signature (remove fileName, file.size, file.type).  So: submit(file, data).  Also, fileName is no longer needed once it's removed here.

File: src/services/formio.service.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+          key,
+          url: `${__S3_BASE_URL__}/${key}`,
+          acl: 'private',
+          size,
+          type
+        }]
+      }
+    };
+
+    $.post(url, payload, null, 'json').done((resp) => alert('success'));
```
Comment (chris-branch) [inline src/services/formio.service.js]: Lets get rid of the alert and navigate to the thankyou.html on successful submission.  But let's only do that if both the form and image are submitted successfully.

File: src/home/home.js (hunk 1 above)
Comment (Ganonside) [reply src/home/home.js]: done

File: src/services/formio.service.js (hunk 2 above)
Comment (Ganonside) [reply src/services/formio.service.js]: done

**Pull request merged**